projects/core/src/user/store/effects/user-details.effect.ts

```diff
@@ -37,12 +46,15 @@ export class UserDetailsEffects {
       this.occUserService
         .updateUserDetails(payload.username, payload.userDetails)
         .pipe(
-          map(
-            _ =>
-              new fromUserDetailsAction.UpdateUserDetailsSuccess(
-                payload.userDetails
-              )
-          ),
+          switchMap(_ => [
+            new fromUserDetailsAction.UpdateUserDetailsSuccess(
+              payload.userDetails
+            ),
+            new AddMessage({
+              text: 'Personal details successfully updated',
+              type: GlobalMessageType.MSG_TYPE_CONFIRMATION,
+            }),
```

Comment:
Even though we have some effects that display messages, I believe it's not an effect's job to display UI messages.

Let's let the caller ( here the UI component ) decide what to do with the success of the process.

This provides more focus about the effect is supposed to do and simplifies testing as well.

Reply:
Fixed